server/api.py

```diff
@@ -4,81 +4,78 @@
 import json
 
 api = Blueprint("api", __name__)
-
 CORS(api)
 
+api.secret_key = "ABC"
+
 
-@api.route('/mentor/signup', methods=['POST'])
+@api.route("/mentor/signup", methods=["POST"])
 def mentor_signup():
     """ Adds new user to User table and new mentor to mentor table """
     
     form_data = request.get_json()
     new_user = create_user(form_data, "mentor")
-
     # add new mentor to Mentor table
     new_company = form_data["company"]
     new_knowledge = form_data["knowledge"]
-
     new_mentor = Mentor(user_id=new_user.id, company=new_company, knowledge=new_knowledge)
-
+    
     db.session.add(new_mentor)
     db.session.commit()
-    
-    return json.dumps({'success':True}), 200, {'ContentType':'application/json'}
+  
+    return jsonify({"success":True, "user_id" : new_user.id}), 200, 
         
-@api.route('/mentee/signup', methods=['POST'])
+@api.route("/mentee/signup", methods=["POST"])
 def mentee_signup():
     """ Adds new user to User table and new mentee to mentor table """
 
     form_data = request.get_json()
     new_user = create_user(form_data, "mentee")  
-
     # add mentee to Mentee table
     new_need_help = form_data["need_help"]
     new_mentee = Mentee(user_id=new_user.id, need_help=new_need_help)
     db.session.add(new_mentee)
     db.session.commit()
 
-    return jsonify({'success':True}), 200
+    return jsonify({"success":True, "user_id" : new_user.id}), 200
```

Comment:
suggestion: because you are sending back a user id, is it possible to create a session for logging in

Reply:
I had session before and I created a simple secret_key just to test and run, but I kept getting runtimeError saying that there was no key set. So I just went around it just to have to route working at first. I need to read more about it. Not sure why that happened, I don't know if it's related to the Blueprint configuration. I'm having problems with simple things that I haven't had before.